tests/network/upgrade/conftest.py

```
@@ -112,3 +143,222 @@ def running_vm_with_bridge(
+    yield from create_ns(
+        admin_client=admin_client,
+        unprivileged_client=unprivileged_client,
+        name="upgrade-localnet-ns",
+        labels=LOCALNET_TEST_LABEL,
+    )
+
+
+@pytest.fixture(scope="session")
+def cudn_localnet_upgrade(
+    admin_client: DynamicClient,
+    nncp_localnet_upgrade: libnncp.NodeNetworkConfigurationPolicy,
```

Ruff (0.15.21)

[warning] Unused function argument: `nncp_localnet_upgrade`

(ARG001)